web/templates/personal_menu.hbs

```diff
@@ -16,16 +16,7 @@
         </header>
         <section class="dropdown-menu-list-section personal-menu-actions" data-user-id="{{user_id}}">
             <ul class="popover-menu-outer-list">
-                {{#if user_time}}
```

Comment:
user_time is not used elsewhere in this template. We should not pass it to the template in that case.

Reply:
understood!. Will create pull request today